app/components/subcases/subcase-item.js

```diff
@@ -89,6 +95,22 @@ export default class SubcaseItemSubcasesComponent extends Component {
     this.subcaseDocuments = sortPieces(pieces);
   }
 
+  @task
+  *loadSubcasePhases() {
+    this.phases = yield this.subcasesService.getSubcasePhases(this.args.subcase);
```

Comment:
This loading task replaces a call to [`subcase.phases`](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/d9b7831ae7494bf5e052d8a62b02fe82c64cb699/app/models/subcase.js#L57).